recipe/meta.yaml

```
@@ -38,7 +39,9 @@ outputs:
         - scipy >=1.7
         - pandas >=1.2
       run_constrained:
-        - {{ pin_subpackage('seaborn', exact=True) }}        
+        # should be {{ pin_subpackage("seaborn", exact=True) }}
+        # but this seems to be broken right now: https://github.com/conda/conda-build/issues/4415
+        - seaborn ={{ version }}=*_{{ build }}
```

```suggestion
        - seaborn {{ version }} *_{{ build }}
```

Not sure if we need the equal separators. This should work too, but better double check locally. The (awfully hacky) way would be to manually modify the repodata.json in the cache and see if this syntax in the `constrains` field works correctly.

We can also patch the repodata if needed.

---

Ah I see in https://github.com/conda/conda-build/issues/4415 that your proposal works so go for it.

---

I will keep mine out of pure laziness 😬